lib/components/Input.js

```diff
@@ -79,12 +83,17 @@ const Input = forwardRef((props, ref) => {
             [errorId]: !!error,
             [helpTextId]: helpText,
           })}
-          data-cy={"input-field"}
+          data-cy="input-field"
           {...otherProps}
           value={value}
           onChange={onChange}
+          data-testid="input-field"
```

Comment:
We dont need the testid here, since the input element can be found by the label.